src/main/service/GeocodingService.ts

```diff
@@ -0,0 +1,40 @@
+import nodeGeocoder, { Entry, Geocoder, Options } from 'node-geocoder';
+import { GeocodingObject } from '../data/GeocodingObject';
+import * as dotenv from 'dotenv';
+
+/**
+ * A Node library for geocoding our restaurant addresses
```

Comment:
How come we need to mention Node?

Reply:
I just mentioned it, is it bad to include it?

Reply:
Oh I'm just not too sure what it means for a service class to be a Node library

Reply:
Oh yeah maybe just make it more clear that it uses a Node library under the hood? I think rn it just sounds like it is a Node library